phpunit.xml.dist

```diff
@@ -35,6 +35,8 @@
 
         <report>
             <clover outputFile="build/logs/clover.xml"/>
+            <html outputDirectory="tests/coverage" highLowerBound="80"/>
+            <text outputFile="tests/coverage.txt"/>
```

Comment:
I don't like to put files in `tests/` because it is for test code.
Use `build/` instead.